uiowa/uiowa
# [Bug] Layout builder row column gap

Resolves https://github.com/uiowa/uiowa/issues/6627. 





# How to test

```
ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli two-column-row-gap-test-page
```
- Familiarize yourself with the bug: https://github.com/uiowa/uiowa/issues/6627.
- Compare with prod https://sandbox.prod.drupal.uiowa.edu/two-column-row-gap-test-page.
- Try removing adding spacing through the styles menu on each section and verify that there are no regressions.
